src/k8s/objects/pvc/pvc.js

```diff
@@ -12,7 +13,7 @@ export const buildPvc = ({ generateName, namespace, storageType, size, storageCl
     volumeMode: 'Filesystem',
     resources: {
       requests: {
-        storage: `${size}Gi`,
+        storage: getValidK8SSize(size, units, 'Gi', true),
```

Comment:
`src/k8s/objects/pvc/pvc.js ` should be pretty generic. I would rather send `size` and `unit` from above to here.

And would rather include the conversion here in `src/k8s/requests/v2v/importVmware.js`:

```js
const extImportPvcsPromises = storages.filter(isImportStorage).map(storage =>
    k8sCreate(
      PersistentVolumeClaimModel,
      buildPvc({
        ...storage,
        generateName: storage.name,
        name: undefined,
        namespace,
        units,
      })
    )
  );
```

where it is clear it is because of the import storage type